strawberry/schema/schema_converter.py

```diff
@@ -411,6 +411,19 @@ def is_type_of(obj: Any, _info: GraphQLResolveInfo) -> bool:
     def from_resolver(
         self, field: StrawberryField
     ) -> Callable:  # TODO: Take StrawberryResolver
+        field.default_resolver = self.config.default_resolver  # type: ignore
```

Comment:
we were thinking of fetching the default resolver from `info.schema.config`, do you think it is possible to access the schema from info with this update?

Reply:
I don't understand, where are you trying to access `info.schema`?

Reply:
we aren't using info now, but I was thinking with @BryceBeagle to change `field.default_resolver` to be a property or something on field that gets the default resolver from `info.schema.config.default_resolver` so we don't have to set the default resolver here, which is a bit awkward

Reply:
Right I see. I would just pass the default resolver to `get_result` on the field rather than accessing it through the `info` object. Creating the `info` object is just unnecessary for fields that don't use it.

Reply:
I think we can access it from the original info object, maybe I can attempt a PR after we merge this one 😊